wormhole-connect/src/utils/inProgressTxCache.ts

```diff
@@ -24,9 +24,14 @@ export const getTxsFromLocalStorage = ():
         try {
           return JSON.parse(item);
         } catch (e: any) {
+          // We can get a SyntaxError from JSON.parse
+          // In that case we should debug log and remove the local storage entry completely,
+          // as we can't know which tx within entry causes the problem without parsing it.
           console.log(
             `Error while parsing localStorage item ${LOCAL_STORAGE_KEY}: ${e}`,
           );
+          // Remove item
+          ls.removeItem(LOCAL_STORAGE_KEY);
```

Comment:
Do you think we should also type-check the parsed values and remove the entry if it's not `Array<TransactionLocal>`? Or is it too much guard-railing?